specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -3756,6 +3844,89 @@
               "Desktop"
             ]
           }
+        }
+      }
+    },
+    "SocketAddressMatchConditionParameters": {
+      "description": "Defines the parameters for SocketAddress match conditions",
+      "required": [
+        "operator",
+        "typeName"
+      ],
+      "properties": {
+        "typeName": {
+          "type": "string",
+          "enum": [
+            "DeliveryRuleSocketAddressConditionParameters"
+          ]
+        },
+        "operator": {
+          "description": "Describes operator to be matched",
+          "type": "string",
+          "enum": [
+            "IPMatch"
```

Comment:
from PM spec, we have IPMatch and GeoMatch. Please confirm with @jessie.

Reply:
Added GeoMatch